### ISSUE TITLE
[DOCS] 3006 RC3 instruction error for RHEL9

### ISSUE THREAD

**Issue by tjyang** [saltstack/salt#64014]
**Description**

RHEL9 installation notes is wrong for RHEL9 at https://docs.saltproject.io/salt/install-guide/en/latest/topics/release-candidate.html

```
sudo rpm --import https://staging.repo.saltproject.io/salt_rc/salt/py3/redhat/9/x86_64/minor/latest/SALT-PROJECT-GPG-PUBKEY-2023.pub
curl -fsSL https://staging.repo.saltproject.io/salt_rc/salt/py3/redhat/9/x86_64/minor/latest/ | sudo tee /etc/yum.repos.d/salt.repo
echo 'baseurl=https://staging.repo.saltproject.io/salt_rc/salt/py3/redhat/$releasever/$basearch/minor/latest' | sudo tee /etc/yum.repos.d/salt.repo
```

**Suggested Fix**

```
sudo rpm --import https://staging.repo.saltproject.io/salt_rc/salt/py3/redhat/9/x86_64/minor/latest/SALT-PROJECT-GPG-PUBKEY-2023.pub
curl -fsSL https://staging.repo.saltproject.io/salt_rc/salt/py3/redhat/9/x86_64/minor/3006.0rc3.repo  | sudo tee /etc/yum.repos.d/salt.repo
```

**Type of documentation**
This could be Salt documentation, Salt modules, the Salt Repo or the Getting Started guide.

**Location or format of documentation**
Insert page URL if applicable.

**Additional context**

**Comment by barbaricyawps** [comment]
@dmurphy18 , can you confirm?

**Comment by dmurphy18** [comment]
@barbaricyawps Yes, but I thought we caught that 'minor/latest' should be 'latest' in the last review of the doc., must have skipped it in the fixes and missed in final review.

Nope, missed it, after catching it initially, sorry

**Comment by barbaricyawps** [comment]
Well, I changed it but then the broken link checker flagged it, so I figured RHEL was different. Apparently not?

**Comment by ScriptAutomate** [comment]
MR opened by @barbaricyawps to address this:
- https://gitlab.com/saltstack/open/docs/salt-install-guide/-/merge_requests/95

**Comment by dmurphy18** [comment]
Left comment in that RC part on the Install Guide, maybe it was just the preview of the guide in that MR that was an issue ?

**Comment by barbaricyawps** [comment]
Latest MR: https://gitlab.com/saltstack/open/docs/salt-install-guide/-/merge_requests/97

**Comment by barbaricyawps** [comment]
This issue is resolved now.
